src/NuGet.Services.EndToEnd/Support/Clients/GalleryClient.cs

```diff
@@ -156,6 +156,30 @@ public async Task UnlistAsync(string id, string version, ITestOutputHelper logge
             await SendToPackageAsync(HttpMethod.Delete, id, version, logger);
         }
 
+        public async Task SearchPackageODataV2FromDBAsync(string id, bool semver2, ITestOutputHelper logger)
```

Comment:
Why is this not version specific?

Why is `semver2` an option here? Can't we just always use `semVerLevel=2.0.0`

Reply:
I removed the semVerLevel.